libnss_ato.c

```diff
@@ -27,6 +27,26 @@
 #include <string.h>
 #include <stdio.h>
 
+/*
+ * Match to the list of process names for which this module should return a
+ * result.
+ * They are processes which call nss_ato on login, and user-run OS command line
+ * programs.
+ */
+const char *VALID_PROC_NAMES[] = {"sshd",
+                                  "login",
+                                  "w"};
```

Comment:
Also would it be better if this list were loaded from a config file rather than hard-coded?